Dockerfile-development

```diff
@@ -5,11 +5,20 @@ RUN \
     mv tini /usr/local/bin/tini && \
     chmod +x /usr/local/bin/tini
 
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
+RUN \
+    mkdir -p /usr/src/skygear && \
+    mkdir -p /usr/src/app
+
+COPY . /usr/src/skygear
 
-COPY . /usr/src/app
-RUN (cd /usr/src/app; npm install && npm link)
+RUN \
+  cd /usr/src/skygear && \
+  npm install && \
+  npm run prepublish && \
+  npm install -g $PWD
```

Comment:
This Dockerfile is aimed for skygear contributor, and the previous code allow development without rebuilding Docker image every step of the way